code/modules/client/preference_setup/loadout/loadout_head.dm

```diff
@@ -444,3 +450,27 @@ var/datum/gear_tweak/hair_block/gear_tweak_hair_block = new()
 			H.flags_inv |= BLOCKHEADHAIR
 		if("No")
 			H.flags_inv &= ~BLOCKHEADHAIR
+
+/*
+	Head Layer Adjustment
+*/
+var/datum/gear_tweak/head_layer/gear_tweak_head_layer = new()
+
+/datum/gear_tweak/head_layer/get_contents(var/metadata)
+	return "Head Layer: [metadata] Suit"
+
+/datum/gear_tweak/head_layer/get_default()
+	return "Over"
+
+/datum/gear_tweak/head_layer/get_metadata(var/user, var/metadata)
+	return input(user, "Choose whether you want the headwear to go over or under the suit.", "Head Layer", metadata) as anything in list("Over", "Under")
```

Comment:
I'd use either "your uniform" or "your jacket" here depending on what you mean, because it can be confusing here